chapter_pages/berlin.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Berlin Local
+permalink: /berlin/
+---
+<style>h1, .main-wrapper h2, h3 {text-align: left; font-weight: bold;}</style>
+# Berlin Tech Workers Coalition
+We first gathered on 20 June 2019. We have been busy organising since. Join our listserv to find out about future meetings and events! We are still figuring out the best ways to broadcast future events.     
+
+## Frequently Asked Questions
+- We are a coalition of workers in and around the tech industry, labour organisers, community organisers, and friends.
+- Our meetings currently are only in English, however we are working on expanding documentation in other languages as well as hosting events in German
+
+## Connect With Us
+- Join our listserv: [TechWorkersBerlin-subscribe@lists.riseup.net](mailto:techworkersberlin-subscribe@lists.riseup.net)
+- Twitter: [@TechWorkersBER](https://twitter.com/TechWorkersBER)
+
+## Accessibility
+We understand accessibility in a broad sense and strive to do better. This includes things like wheelchair accessibility, smoke free environments, short(er) meetings and child care.  
```

Comment:
How about 'We try to host meetings that are' instead of 'This includes things like'?